prow/scripts/provision_vm_and_start_kyma.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -o errexit
+
+cleanup() {
+    ARG=$?
+    echo "
+################################################################################
+# Removing instance kyma-integration-test-${RANDOM_ID}
+################################################################################
+    "
+
+    gcloud compute instances delete kyma-integration-test-${RANDOM_ID}
+    exit $ARG
+}
+
+readonly SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck disable=SC1090
+source "${SCRIPT_DIR}/library.sh"
+
+RANDOM_ID=$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 8 | head -n 1)
+
+LABELS=""
+if [[ -z "${PULL_NUMBER}" ]]; then
+    LABELS="--labels branch=$PULL_BASE_REF,job-name=kyma-integration"
+else
+    LABELS="--labels pull-number=$PULL_NUMBER,job-name=kyma-integration"
+fi
+
+echo "
+################################################################################
+# Creating a new instance named kyma-integration-test-${RANDOM_ID}
+################################################################################
+"
+
+gcloud compute instances create kyma-integration-test-${RANDOM_ID} \
+    --metadata enable-oslogin=TRUE --image debian-9-stretch-v20181011 \
+    --image-project debian-cloud --machine-type n1-standard-4 --boot-disk-size 20 $LABELS
+
+trap cleanup exit
+
+echo "
+################################################################################
+# Copying Kyma to the instance
+################################################################################
+"
+
+for i in $(seq 1 5); do
+    [ $i -gt 1 ] && echo 'Retrying in 15 seconds..' && sleep 15;
+    gcloud compute scp --quiet --recurse /home/prow/go/src/github.com/kyma-project/kyma kyma-integration-test-${RANDOM_ID}:~/kyma && break;
```

Comment:
why do you introduce retries mechanism? How often we have problem with that command?

Reply:
Network error occurs rarely but still leads to the test failure and we can prevent it with a simple retry. So, why not have it?